rust-grammar-dpdfa/grammar.rs

```diff
@@ -0,0 +1,496 @@
+fn vis() {
```

Comment:
We want to check that this file is formatted correctly.

- Update the justfile
- Update the CI file